LeaderboardBackend/Models/Requests/ParticipationRequests.cs

```diff
@@ -18,3 +18,10 @@ public class CreateParticipationRequest
 	public bool IsSubmitter { get; set; } = true;
 }
 
+public class UpdateParticipationRequest
+{
+	public string Comment { get; set; } = "";
+
+	[Required(ErrorMessage = "Please add a VoD link. Your participation can't be confirmed otherwise.")]
```

Comment:
Maybe we should make a custom rule for this such that it's both required and that it has to be a link to a valid video, or a link from a set list of domains.

Reply:
We could just add a FIXME on this, for a future PR to tackle

Reply:
Yeah probably out of scope, but I can add a FIXME comment

Reply:
I'll do it